### PR TITLE
rename to credential_configuration_id

### PR DESCRIPTION
**Pull request by Sakurann**
applying three changes:
- In the credential issuer metadata: credentials_supported -> credential_configurations_supported
- in the credential offer: credentials -> credential_configurations_offered
- In the authorization details: credentials_supported_id -> credential_configuration_id

this PR needs to be merged before #137

### CODE REVIEW

**Comment by Sakurann** [comment]
there is some diff not present in Paul's original PR, which i think is caused by paul's branch not being up to date with main. should be fixed in this PR when paul's branch I am pushing to is up to date with main.

**Comment by Sakurann** [comment]
I think it is ok for this PR to be merged before pr #155 is merged, to prevent breaking changes post ID-1

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -1286,7 +1286,7 @@ This specification also defines a new OAuth 2.0 Authorization Server metadata [@
 
 Credential Issuers often want to know what Wallet they are issuing Credentials to and how private keys are managed for the following reasons:
 
-* The Credential Issuer MAY want to ensure that private keys are properly protected from exfiltration and replay to prevent an adversary from impersonating the legitimate Credential holder by presenting her Credential.
+* The Credential Issuer MAY want to ensure that private keys are properly protected from exfiltration and replay to prevent an adversary from impersonating the legitimate Credential Holders by presenting their Credentials.
```
Comment (danielfett) [inline openid-4-verifiable-credential-issuance-1_0.md:1289]:
```suggestion
* The Credential Issuer MAY want to ensure that private keys are properly protected from exfiltration and replay to prevent an adversary from impersonating the legitimate Credential Holder by presenting their Credentials.
```
Not sure, but I think we usually talk about a single Holder in these kind of statements.

**Review by danielfett — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ For security considerations, see (#credential-offer-security).
...
-* `credentials`: REQUIRED. An array of unique strings that each identify one of the keys in the name/value pairs stored in the `credentials_supported` Credential Issuer metadata property. The Wallet uses this string value to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, this string value can be used to obtain `scope` value to be used in the Authorization Request.
+* `credential_configurations_offered`: REQUIRED. An array of unique strings that each identify one of the keys in the name/value pairs stored in the `credential_configurations_supported` Credential Issuer metadata property. The Wallet uses this string value to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, this string value can be used to obtain `scope` value to be used in the Authorization Request.
```
Comment (peppelinux) [inline openid-4-verifiable-credential-issuance-1_0.md:326]:
```suggestion
* `credential_configurations_offered`: REQUIRED. An array of unique strings, each representing a key in the `credential_configurations_supported` property of the Credential Issuer metadata. The Wallet utilizes these string values to access the corresponding object that holds information about the offered Credential, as defined in (#credential-issuer-parameters). For instance, these string values can be used to retrieve the scope value for use in the Authorization Request.
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -413,7 +413,7 @@ There are two possible ways to request issuance of a specific Credential type in
...
 
 * `type`: REQUIRED. String that determines the authorization details type. MUST be set to `openid_credential` for the purpose of this specification.
-* `credentials_supported_id`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credentials_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credentials_supported` map conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
+* `credential_configuration_id`: REQUIRED. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```
Comment (peppelinux) [inline openid-4-verifiable-credential-issuance-1_0.md:416]:
```suggestion
* `credential_configuration_id`: REQUIRED. A string that uniquely identifies the Credential defined in the `credential_configurations_supported` map within the Credential Issuer Metadata, as detailed in (#credential-issuer-parameters). The corresponding object in the `credential_configurations_supported` map provides specific information, such as format, for the issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 4]
```diff
@@ -704,7 +704,7 @@ For cryptographic binding, the Client has the following options to provide crypt
...
-* `proof`: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to.  The `proof` object is REQUIRED if the `proof_types` parameter is non-empty and present in the `credentials_supported` map of the issuer metadata for the requested credential. The `proof` object MUST contain a following claim:
+* `proof`: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to.  The `proof` object is REQUIRED if the `proof_types` parameter is non-empty and present in the `credential_configurations_supported` map of the issuer metadata for the requested credential. The `proof` object MUST contain a following claim:
```
Comment (peppelinux) [inline openid-4-verifiable-credential-issuance-1_0.md:707]:
```suggestion
* `proof`: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to.  The `proof` object is REQUIRED if the `proof_types` parameter is non-empty and present in the `credential_configurations_supported` map of the Issuer metadata for the requested Credential. The `proof` object MUST contain a following claim:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 5]
```diff
@@ -1246,9 +1246,9 @@ This specification defines the following Credential Issuer Metadata:
...
-* `credentials_supported`: REQUIRED. An object that describes specifics of the Credential that the Credential Issuer supports issuance of. This object contains a list of name/value pairs, where each name is a unique identifier of the supported Credential being described. This identifier is used in the Credential Offer as defined in (#credential_offer_parameters) to communicate to the Wallet which Credential is being offered. The value is an object that contains metadata about specific Credential and contains the following parameters defined by this specification:
+* `credential_configurations_supported`: REQUIRED. An object that describes specifics of the Credential that the Credential Issuer supports issuance of. This object contains a list of name/value pairs, where each name is a unique identifier of the supported Credential being described. This identifier is used in the Credential Offer as defined in (#credential_offer_parameters) to communicate to the Wallet which Credential is being offered. The value is an object that contains metadata about specific Credential and contains the following parameters defined by this specification:
```
Comment (peppelinux) [inline openid-4-verifiable-credential-issuance-1_0.md:1249]:
```suggestion
* `credential_configurations_supported`: REQUIRED. An object detailing the specifics of the Credential that the Credential Issuer is capable of issuing. This object comprises a list of name/value pairs, where each name uniquely identifies the supported Credential being described. This identifier is utilized in the Credential Offer, as outlined in (#credential_offer_parameters), to inform the Wallet about the offered Credential. The value is an object that contains metadata about a specific Credential and includes the following parameters defined by this specification:
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 6]
```diff
@@ -1336,7 +1336,7 @@ The Wallet is supposed to detect signs of fraudulent behavior related to the Cre
 
 If an adversary is able to get hold of a key proof defined in (#proof_types), the adversary could get a Credential issued that is bound to a key pair controlled by the victim.
 
-Note: For the attacker to be able to present to the Verifier a Credential bound to a replayed Key Proof, the attacker also needs to obtain the victim's private key. To limit this, servers are RECOMMENDED to check how the Wallet protects the private keys, using mechanisms such as Key Based Client Authentication defined in [@I-D.looker-oauth-attestation-based-client-auth].
+Note: For the attacker to be able to present to the Verifier a Credential bound to a replayed Key Proof, the attacker also needs to obtain the victim's private key. To limit this, servers are RECOMMENDED to check how the Wallet protects the private keys, using mechanisms such as Key Based Client Authentication defined in [@!I-D.ietf-oauth-attestation-based-client-auth].
```
Comment (peppelinux) [inline openid-4-verifiable-credential-issuance-1_0.md:1339]:
```suggestion
Note: For the attacker to be able to present to the Verifier a Credential bound to a replayed Key Proof, the attacker also needs to obtain the victim's private key. To limit this, servers are RECOMMENDED to check how the Wallet protects the private keys, using mechanisms such as Attestation-Based Client Authentication defined in [@!I-D.ietf-oauth-attestation-based-client-auth].
```

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 3 above)
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:416]:
```suggestion
* `credential_configuration_id`: REQUIRED. String specifying the Credential Configuration Identifier that of the Credential being described in the `credential_configurations_supported` Credential Issuer Metadata parameter as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` parameter conveys the details, e.g. format, for the requested Credential issuance. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 7]
```diff
@@ -448,7 +448,7 @@ Note: Applications MAY combine authorization details of type `openid_credential`
 
 In addition to a mechanism defined in (#credential-authz-request), Credential Issuers MAY support requesting authorization to issue a Credential using OAuth 2.0 scope parameter.
 
-When the Wallet does not know which scope value to use to request issuance of a certain Credential, it can discover it using the `scope` Credential Issuer metadata parameter defined in (#credential-issuer-parameters). When the flow starts with a Credential Offer, the Wallet can use the `credentials` parameter values to identify object(s) in the `credentials_supported` map in the Credential Issuer metadata parameter and use `scope` parameter value from that object.
+When the Wallet does not know which scope value to use to request issuance of a certain Credential, it can discover it using the `scope` Credential Issuer metadata parameter defined in (#credential-issuer-parameters). When the flow starts with a Credential Offer, the Wallet can use the `credentials_configurations_offered` parameter values to identify object(s) in the `credential_configurations_supported` map in the Credential Issuer metadata parameter and use `scope` parameter value from that object.
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:451]:
```suggestion
When the Wallet does not know which scope value to use to request issuance of a certain Credential, it can discover it using the `scope` Credential Issuer metadata parameter defined in (#credential-issuer-parameters). When the flow starts with a Credential Offer, the Wallet can use the Credential Configuration Identifiers in the `credentials_configurations_offered` parameter to identify the Credential Configurations in the `credential_configurations_supported` Credential Issuer metadata parameter and use the `scope` parameter value from that object.
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 8]
```diff
@@ -1723,31 +1723,143 @@ TBD
...
+
+## OAuth Dynamic Client Registration Metadata Registry
+
+This specification registers the following client metadata name in the IANA "OAuth Dynamic Client Registration Metadata" registry [@!IANA.OAuth.Parameters] established by [@!RFC7591].
+
+* Client Metadata Name: credential_offer_endpoint
+* Client Metadata Description: Credential Offer Endpoint
+* Change Controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Reference: (#credential_offer_endpoint) of this specification
+
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:1808]: These should be removed since we have a separate PR that adds the IANA entries.

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 9]
```diff
@@ -1723,31 +1723,143 @@ TBD
...
+  - Magic number(s): n/a
+  - File extension(s): n/a
+  - Macintosh file type code(s): n/a
+* Person & email address to contact for further information: Torsten Lodderstedt, torsten@lodderstedt.net
+* Intended usage: COMMON
+* Restrictions on usage: none
+* Author: Torsten Lodderstedt, torsten@lodderstedt.net
+* Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
+* Provisional registration? No
+
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:1862]: Shouldn't these get removed since we have a separate PR for IANA registration?

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 10]
```diff
@@ -269,7 +269,7 @@ The following diagram is based on the Credential Issuer initiated flow, as illus
         |   interacts    |                                                    |
         |--------------->|                                                    |
         |                |                                                    |
-        |                |  (4) Token Request (Pre-Authorized Code, pin)      |
+        |                |  (4) Token Request (Pre-Authorized Code, tx_code)  |
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:272]: Should this be in a separate PR?

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 2 above)
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:326]:
```suggestion
* `credential_configurations_offered`: REQUIRED. An array of unique strings that each identify one of the Credential Configuration Identifiers in the `credential_configurations_supported` Credential Issuer metadata property. The Wallet uses this string value to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, this string value can be used to obtain `scope` value to be used in the Authorization Request.
```

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 11]
```diff
@@ -602,14 +602,14 @@ grant_type=urn:ietf:params:oauth:grant-type:pre-authorized_code
...
+  * `credential_identifiers`: OPTIONAL. Array of strings that each uniquely identify a Credential instance that can be issued using Access Token returned in this response. Each Credential instance is a unique Credential described using the same entry in the `credential_configurations_supported` Credential Issuer metadata, but can contain different claim values or different subset of claims within the claimset identified by the Credential type. This parameter can also be used to simplify the Credential Request, since as defined in (#credential_request) `credential_identifier` parameter replaces `format` and any other Credential format specific parameters in the Credential Request. When received, the Wallet MUST use these values together with an Access Token in the subsequent Credential Request(s).
```
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:612]: I still don't understand the difference and I think it is confusing since it conflates two concepts but this is probably just me. I hope the terminology PR will fix my lack of knowledge.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 4 above)
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:707]:
```suggestion
* `proof`: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to.  The `proof` object is REQUIRED if the `proof_types` parameter is non-empty and present in the `credential_configurations_supported` Credential Issuer metadata for the requested credential. The `proof` object MUST contain a following claim:
```

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 5 above)
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:1249]:
```suggestion
* `credential_configurations_supported`: REQUIRED. An object that describes specifics of the Credential that the Credential Issuer supports issuance of. This object contains a list of name/value pairs, where each name is a unique Credential Configuration Identifier of the supported Credential being described. This identifier is used in the Credential Offer as defined in (#credential_offer_parameters) to communicate to the Wallet which Credential is being offered. The value is an object that contains metadata about specific Credential and contains the following parameters defined by this specification:
```

**Review by awoie — CHANGES_REQUESTED** [review]
IMO, the IANA part and the tx_code change should get removed from this PR. It would be nice to get the Credential Configuration Identifier language in there if the suggested language makes sense. Otherwise looks good to me.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 10 above)
Comment (paulbastian) [reply openid-4-verifiable-credential-issuance-1_0.md:272]: I think this is a left-over, might be addressed by existing PRs

**Review by paulbastian — APPROVED** [review]
I approve the changes, having clear temrinology is a long-term benefit fot the specification and before ID-1 is the best time.
I also agree with Oliver, keeping the PIN and IANA section out of this PR, as it is already very large.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 11 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:612]: @Sakurann need to fix few sentences here to clarify that there is no polymorphism. "the identifier refers to the instances of the credential configuration". drop `also`.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 2 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:326]:
```suggestion
* `credential_configurations_offered`: REQUIRED. An array of unique strings that each identify one of the Credential Configuration Identifiers in the `credential_configurations_supported` Credential Issuer metadata property. The Wallet uses these string values to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, these string values can be used to obtain `scope` value to be used in the Authorization Request.
```

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 3 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:416]:
```suggestion
* `credential_configuration_id`: REQUIRED. String specifying the Credential Configuration Identifier that of the Credential being described in the `credential_configurations_supported` Credential Issuer Metadata parameter as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` parameter conveys the details, such as format, for the issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format_profiles).
```

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 7 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:451]: please introduce a new term "credential configuration identifier" in PR #155. this PR is strictly meant to change the parameter names only.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 5 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1249]: please introduce a new term "credential configuration identifier" in PR #155. this PR is strictly meant to change the parameter names only.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 9 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1862]: this happened because the branch used in Paul's PR was not up to date with main. will make sure there are no conflicts when merging.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 8 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1808]: this happened because the branch used in Paul's PR was not up to date with main. will make sure there are no conflicts when merging.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 10 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:272]: this happened because the branch used in Paul's PR was not up to date with main. will make sure there are no conflicts when merging.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 11 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:612]:
```suggestion
  * `credential_identifiers`: OPTIONAL. Array of strings that each uniquely identify a Credential that can be issued using Access Token returned in this response. Each Credential is described using the same object in the `credential_configurations_supported` Credential Issuer metadata, but can contain different claim values or different subset of claims within the claimset identified by that Credential type. This parameter can be used to simplify the Credential Request, since, as defined in (#credential_request), `credential_identifier` parameter replaces `format` and any other Credential format specific parameters in the Credential Request. When received, the Wallet MUST use these values together with an Access Token in the subsequent Credential Request(s).
```

@awoie @tlodderstedt please check if this is an improvement

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 3 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md]: please do the bikeshedding in PR https://github.com/openid/OpenID4VCI/pull/155. this PR is strictly meant to change the parameter names only.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 5 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md:1249]: please do the bikeshedding in PR https://github.com/openid/OpenID4VCI/pull/155. this PR is strictly meant to change the parameter names only.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 4 above)
Comment (Sakurann) [reply openid-4-verifiable-credential-issuance-1_0.md]: please do the bikeshedding in PR https://github.com/openid/OpenID4VCI/pull/155. this PR is strictly meant to change the parameter names only.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 7 above)
Comment (awoie) [inline openid-4-verifiable-credential-issuance-1_0.md:451]:
```suggestion
When the Wallet does not know which scope value to use to request issuance of a certain Credential, it can discover it using the `scope` Credential Issuer metadata parameter defined in (#credential-issuer-parameters). When the flow starts with a Credential Offer, the Wallet can use the `credential_configurations_offered` parameter values to identify object(s) in the `credential_configurations_supported` map in the Credential Issuer metadata parameter and use `scope` parameter value from that object.
```

Comment (awoie) [reply openid-4-verifiable-credential-issuance-1_0.md:451]: fixed typo

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 11 above)
Comment (awoie) [reply openid-4-verifiable-credential-issuance-1_0.md:612]:
```suggestion
  * `credential_identifiers`: OPTIONAL. Array of strings, each uniquely identifying a Credential that can be issued using the Access Token returned in this response. Each of these Credentials corresponds to the same entry in the `credential_configurations_supported` Credential Issuer metadata but can contain different claim values or a different subset of claims within the claimset identified by that Credential type. This parameter can be used to simplify the Credential Request, as defined in (#credential_request), where the `credential_identifier` parameter replaces the format and any other Credential format-specific parameters in the Credential Request. When received, the Wallet MUST use these values together with an Access Token in subsequent Credential Requests.
```

**Review by awoie — APPROVED** [review]
Looks good now. Please commit my suggestions before merging.

File: examples/credential_issuer_metadata_jwt_vc_json.json [hunk 12]
```diff
@@ -14,7 +14,7 @@
             "locale": "fr-FR"
         }
     ],
-    "credentials_supported": {
+    "credential_configurations_supported": {
```
Comment (tlodderstedt) [inline examples/credential_issuer_metadata_jwt_vc_json.json:17]: I think `credential_configurations` is enough (as this is a credential offer already).

**Review by tlodderstedt — APPROVED** [review]
ok with me, however I would suggest to use `credential_configurations` instead of `credential_configurations_offered` as the parameter is used inside the credential offer. The name seems too much to me (but no hill to die on).

**Review by c2bo — APPROVED** [review]
I agree with @tlodderstedt  that `credential_configurations ` might be a better/easier name - apart from that it looks fine imho

**Comment by Sakurann** [comment]
good point - renamed to `credential_configurations`

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 13]
```diff
@@ -602,14 +602,14 @@ grant_type=urn:ietf:params:oauth:grant-type:pre-authorized_code
...
+  * `credential_identifiers`: OPTIONAL. Array of strings, each uniquely identifying a Credential that can be issued using the Access Token returned in this response. Each of these Credentials corresponds to the same entry in the `credential_configurations_supported` Credential Issuer metadata but can contain different claim values or a different subset of claims within the claimset identified by that Credential type. This parameter can be used to simplify the Credential Request, as defined in (#credential_request), where the `credential_identifier` parameter replaces the format and any other Credential format-specific parameters in the Credential Request. When received, the Wallet MUST use these values together with an Access Token in subsequent Credential Requests.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:612]:
```suggestion
  * `credential_identifiers`: OPTIONAL. Array of strings, each uniquely identifying a Credential that can be issued using the Access Token returned in this response. Each of these Credentials corresponds to the same entry in the `credential_configurations_supported` Credential Issuer metadata but can contain different claim values or a different subset of claims within the claimset identified by that Credential type. This parameter can be used to simplify the Credential Request, as defined in (#credential_request), where the `credential_identifier` parameter replaces the `format` parameter and any other Credential format-specific parameters in the Credential Request. When received, the Wallet MUST use these values together with an Access Token in subsequent Credential Requests.
```

**Comment by Sakurann** [comment]
this PR addresses renaming part of issue #91
